Misc/NEWS.d/next/Library/2018-11-15-07-14-32.bpo-35226.wJPEEe.rst

```diff
@@ -0,0 +1 @@
+Fix equality for nested unittest.mock.call objects.
```

Comment:
I'd try to be more explicit here. 

```Recursively check  arguments when testing for equality of :class:`unittest.mock.call` objects.```.
As the parent names are already being checked via the name attribute.

Reply:
Happy to polish this once we get some consensus on this PR.